third_party/outline-ss-server/Makefile

```diff
@@ -1,17 +1,23 @@
 VERSION=1.4.0
+SHA256_linux_x86_64=f51bcb6391cca0ae828620c429e698a3b7c409de2374c52f113ca9a525e021a8
+SHA256_linux_aarch64=14ae581414c9aab04253a385ef1854c003d09f545f6f8a3a55aa987f0c6d3859
+APP_ARCH_linux_x86_64=x86_64
+APP_ARCH_linux_aarch64=arm64
 
 .PHONY: all
 all: bin/linux/outline-ss-server bin/macos/outline-ss-server
 
 bin/linux/outline-ss-server: OS=linux
```

Comment:
Could you make it so that the output path looks like `bin/linux_x86_64/outline-ss-server`?
I don't wanna risk accidentally having a file for the wrong architecture there.

That will allow you to have the code below, which is easier to understand:

```makefile
bin/linux_x86_64/outline-ss-server: SHA256=f51bcb6391cca0ae828620c429e698a3b7c409de2374c52f113ca9a525e021a8
bin/linux_x86_64/outline-ss-server: APP_ARCH=x86_64
``` 

Perhaps you use pattern matching? (https://www.gnu.org/software/make/manual/html_node/Pattern-Match.html).

```Makefile
bin/linux_%/outline-ss-server: APP_ARCH=%
``` 

Same for prometheus and macos.

Reply:
Thanks for the good idea. Fixed